pgmpy/tests/test_utils/test_utils.py

```diff
@@ -254,3 +255,17 @@ def test_model_categorization(self):
 
         cont_model = get_example_model("magic-irri")
         self.assertIsInstance(cont_model, LinearGaussianBayesianNetwork)
+
+    def test_invalid_state_name(self):
```

Comment:
I think a more suitable place for these tests would be in: https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/tests/test_inference/test_ExactInference.py#L13 . Could you please move these?

Reply:
Thanks for the kind suggestion! Fixed in commit 3b24eb72c401811bdd600c9b41eec26ad21491e1